langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -263,6 +373,10 @@ public isolated function fromCodePointInts(int[] codePoints) returns string|erro
 # An int is a valid code point if it is in the range 0 to 0x10FFFF inclusive,
 # but not in the range 0xD800 or 0xDFFF inclusive.
 #
+# ```ballerina
+# string:Char charValue = check string:fromCodePointInt(97);
```

Comment:
Shall we use a variable as the argument?

Reply:
Any reason for that? Is that how we handled in other places? I feel it is a bit too much.

Reply:
When this function is actually used, wouldn't it most likely be with a variable? If it is a known code point, we could probably directly use the character?

Reply:
Yes, in real-world function argument values are going to be variable references (they cannot be hardcoded values). Just because of that, I don't think we should move all these argument values to separate variables and use those references as function arguments. Sorry, that doesn't feel correct to me. Most of our langlib functions need arguments. Are we going to move all those argument values to variables? It looks unnecessary and makes the example looks long and complex. Yes, if the argument values are complex or not shorter then it makes sense to me.